ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -98,6 +98,12 @@
     reason="Tests are not running on AWS deployed cluster"
 )
 
+rh_internal_lab_only = pytest.mark.skipif(
+    (config.ENV_DATA['platform'].lower() == 'aws'
```

Comment:
Won't be easier to have only:
config.ENV_DATA['platform'].lower() != "vsphere"
?

Reply:
there will be openstack and having position checks is clearer.